docs/architecture-decisions/adr003-e2e.md

```diff
@@ -0,0 +1,21 @@
+# ADR003: E2E Test Strategy
+
+## Status
+
+accepted
+
+## Context
+
+There is the need to run E2Es automatically in the pipeline to test our whole application. We had no strategy on what and where to do any end-to-end testing.
+
+## Decision
+
+- We are only end-to-end testing our SPA application to reduce complexity.
```

Comment:
Also, E2E testing the API while having an E2E SPA test, which covers all functionality, doesn't add any value. Additionally, API E2E tests make most sense when regression testing multiple API versions. We don't have that.